File: app/src/main/java/com/asmat/rolando/popularmovies/ui/activities/MyListsActivity.java
package com.asmat.rolando.popularmovies.ui.activities;


import com.asmat.rolando.popularmovies.R;
import com.asmat.rolando.popularmovies.ui.adapters.pager.BaseSectionsPagerAdapter;
import com.asmat.rolando.popularmovies.ui.adapters.pager.MyListsPagerAdapter;

public class MyListsActivity extends BaseActivity {

    @Override
    String getActivityTitle() {
        return getString(R.string.my_lists);
    }

    @Override
    BaseSectionsPagerAdapter getPagerAdapter() {
        return new MyListsPagerAdapter(getSupportFragmentManager(),this);
    }
}

File: app/src/main/java/com/asmat/rolando/popularmovies/ui/activities/MyListsActivity.kt
package com.asmat.rolando.popularmovies.ui.activities


import com.asmat.rolando.popularmovies.R
import com.asmat.rolando.popularmovies.ui.adapters.pager.BaseSectionsPagerAdapter
import com.asmat.rolando.popularmovies.ui.adapters.pager.MyListsPagerAdapter

class MyListsActivity : BaseActivity() {

    override val activityTitle: String
        get() = getString(R.string.my_lists)

    override val pagerAdapter: BaseSectionsPagerAdapter
        get() = MyListsPagerAdapter(supportFragmentManager, this)
}

